addons/project/static/src/views/project_project_kanban/project_project_kanban_renderer.js

```diff
@@ -2,11 +2,24 @@
 
 import { KanbanRenderer } from "@web/views/kanban/kanban_renderer";
 import { ProjectProjectKanbanHeader } from "./project_project_kanban_header";
+import { onWillStart } from "@odoo/owl";
+import { user } from "@web/core/user";
 
 
 export class ProjectProjectKanbanRenderer extends KanbanRenderer {
     static components = {
         ...KanbanRenderer.components,
         KanbanHeader: ProjectProjectKanbanHeader,
     };
+
+    setup() {
+        super.setup();
+        onWillStart(async () => {
+            this.isProjectManager = await user.hasGroup('project.group_project_manager');
+        });
+    }
+
+    get canResequenceGroups() {
+        return super.canResequenceGroups && this.isProjectManager;
+    }
```

Comment:
We should think of making this generic, otherwise code duplication will occur. This issue likely arises in Helpdesk, CRM, and other areas as well.